### ISSUE TITLE
change 1-sided fisher exact test to two-sided

### ISSUE THREAD

**Issue by inodb** [cBioPortal/cbioportal#9943]
In both the Mutual Exclusivity tab and the Comparison page/tab, we use 1-side Fisher's Exact test. Two-sided is more appropriate:

- [ ] fix for mutual exclusivity tab
- [ ] fix for comparison tab
- [ ] update faq (see [comment](https://github.com/cBioPortal/cbioportal/pull/10199/files#r1206992259))

Some anonymous feedback on this from others:

- I actually noticed that in the portal it uses the one-sided but in papers, I've seen two-sided and that's what i personally use. You're predicting the direction of the effect with one-sided, something i'm not sure how you can do in this situation.  with directionality (one sided)  the pvalue is half the 2 sided pvalue so more samples will pass FDR cutoffs (in laymans terms). (caveat: I'm not a stats expert )
- I agree with the above, I always use a two-sided test in this setting
- Generally speaking, a two-sided test is almost always more appropriate 

And user that raised it: https://groups.google.com/g/cbioportal/c/WbPzItsK_Vc/m/wlTdzmf3AQAJ

Question: T-test for continuous data (mRNA tab and protein tab), should we use two sided here as well? Maybe create follow-up ticket?

**Comment by Eric-Jiang1** [comment]
working on this issue, will get back to you tomorrow on Slack
